components/Images/images.css

```diff
@@ -0,0 +1,18 @@
+@import "../../style_guide.css";
+
+body {
+  background-color: var(--bg-color);
+  color: var(--txt-color);
+}
+
+.images {
```

Comment:
Why are you restricting it to any size. Image'll always be used in some div so it should talk parent's width or height 100% and maintain aspect ratio.

Reply:
I commented that out right?

Reply:
I refactored the images please check